exchanges/coinbasepro/coinbasepro.go

```diff
@@ -766,9 +766,13 @@ func (c *CoinbasePro) SendAuthenticatedHTTPRequest(ctx context.Context, ep excha
 		n := strconv.FormatInt(time.Now().Unix(), 10)
 		message := n + method + "/" + path + string(payload)
 
+		secret, err := crypto.Base64Decode(creds.Secret)
```

Comment:
I think this should be conditional upon `c.API.CredentialsValidator.RequiresBase64DecodeSecret`

Reply:
Added the condition, perhaps we should add `!b.LoadedByConfig` as well?